### PR TITLE
Fix the Put and Patch parameter order mismatching existing .Net SDK after documentation update

### PR DESCRIPTION
**Pull request by NonStatic2014**
In previous documentation polish, the order of parameters for PUT and PATCH were changed. This PR is trying to fix it to make the new version backward compatible.

### CODE REVIEW

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 1]
```diff
@@ -46,7 +40,13 @@
             "schema": {
               "$ref": "#/definitions/WebService"
             }
-          }                
+          },
```
Comment (amarzavery) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: What you had was correct. ResourceGroupName should be the first parameter then comes Rp specific parameter name and in the end should be the body parameter. That is the general convention. If this was not being followed previously then that was incorrect.

Check this example: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-01-01/swagger/storage.json#L64. subscriptionId and api-version will be client properties in most cases hence their order is not that important.

Comment (amarzavery) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Infact I would suggest - resourceGroupName, webserviceName, bodyParameter, api-version and then subscriptionID as the parameter

**Review by amarzavery — CHANGES_REQUESTED** [review]

**Comment by hovsepm** [comment]
>               "$ref": "#/definitions/PaginatedWebServicesList"

you will need to use x-ms-pagable extension to indicate that your List operation may return several pages of results. 

---
Refers to: arm-machinelearning/2016-05-01-preview/swagger/webservices.json:255 in 1c67a74. [](commit_id = 1c67a74b6846c9b454b77cc6fac7d44b6cdb1e40, deletion_comment = False)

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 2]
```diff
@@ -196,24 +199,27 @@
         "tags": [
           "WebServices"
         ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
         "operationId": "WebServices_ListInResourceGroup",
```
Comment (amarzavery) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: One last change: Please name this operation: WebServices_List~~In~~**By**ResourceGroup. That is the standard convention. Those libraries which are not following this convention, we are asking them to change this in the next version. Consistent naming pattern across RPs provides a good experience to customers in Azure and they don't have to keep learning these unnecessary things.

**Review by amarzavery — APPROVED** [review]

**Pull request merged**